release-notes/7.0/supported-os.md

```diff
@@ -28,13 +28,13 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
-[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS][CentOS]                      | 7                     | x64               | [CentOS][CentOS-lifecycle]
```

Comment:
This one is really called `CentOS Linux`, not just `CentOS`. See https://www.centos.org/cl-vs-cs/

So please keep the name upfront `CentOS`. The lifecycle column can be `CentOS`, since it's referring to the project and is appliable to both versions.